src/command_modules/azure-cli-acs/azure/cli/command_modules/acs/tests/latest/test_aks_commands.py

```diff
@@ -177,12 +184,102 @@ def test_aks_create_service_no_wait(self, resource_group, resource_group_locatio
         ])
 
         # delete
-        self.cmd('aks delete -g {resource_group} -n {name} --yes', checks=[self.is_empty()])
+        self.cmd(
+            'aks delete -g {resource_group} -n {name} --yes', checks=[self.is_empty()])
```

Comment:
Why break at 80? The line limitation we use is 120.

Reply:
Fixed.